defaults/main.yml

```diff
@@ -28,6 +28,8 @@ apache_vhosts_ssl: []
   #   # Optional.
   #   certificate_chain_file: "/path/to/certificate_chain.crt"
 
+apache_ignore_missing_ssl_certificate: True
```

Comment:
Just a style note; though I may change this at some point, for now I'm using lowercase bool definitions in all my roles, so can you update this to `true`?